### PR TITLE
Update pytz and requests to their latest

### PR DESCRIPTION
**Pull request by stephendonner**
Sorry for the previous, errant pull, which had both peepin and hashin changes - this one should be good to go.

### CODE REVIEW

**Comment by stephendonner** [comment]
@bobsilverberg and/or @karlht another r? if you wouldn't mind?

File: requirements.txt [hunk 1]
```diff
@@ -149,24 +149,23 @@ python-memcached==1.53
...
+# sha256: kwkjNzLHu2HFhUc_i4C5q6See03I4CIAfEkm2wofPYQ
+# sha256: 3EPD1Ds15_1TIS5v-Y29_XzOS8mhoRD_gEZk2Ah7xYM
+# sha256: 8dzDSkSpO_HF9y9cdZhaogZePTyp49Cel-3BSRZHNio
+# sha256: OZfBHc5sxG5ypXEUksWf1MMuD3fOzLTO_DG1ToJrDjI
+# sha256: aQU_lRB59cv5REzJv4r5oieEHKX9nLjtTOzxoTl62rs
+# sha256: wZPfoWesMsjLlvJsvNkpclkbIr2gusPv_b2wTebMVdY
+# sha256: NEnaGQUWVdTAu1w3GRMxdIvK0VgE2BZ2qIRR7ymTcKg
+# sha256: pA3dk1zzaD_4AQKLkAGjjqgP16e7plesOA4Ul_hoYe4
+pytz==2016.3
+# sha256: ET-7pVManjSUW302szoIToul0GZLcDyBp8Vy2RkZpbg
```
Comment (bobsilverberg) [inline requirements.txt]: @stephendonner Please include a blank line between packages, as was there before this change.

**Comment by stephendonner** [comment]
@bobsilverberg done.  Added back the two blank lines, recommitted, rebased, and squashed commits into the original commit + message.  r? again?

**Comment by bobsilverberg** [comment]
Looks good, thanks @stephendonner!

**Pull request merged**